config/routes.rb

```diff
@@ -12,6 +12,7 @@
     resources :orders
     resources :line_items
     resources :carts
+    resources :ratings
```

Comment:
use only update 
and restrict to json request only

Reply:
use format: :json